src/Services/FileActivationHandler.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+using DevHome.Activation;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Settings.ViewModels;
+using DevHome.SetupFlow.ViewModels;
+using DevHome.Views;
+using Microsoft.UI.Xaml;
+using Windows.ApplicationModel.Activation;
+using Windows.Storage;
+
+namespace DevHome.Services;
+internal class FileActivationHandler : ActivationHandler<FileActivatedEventArgs>
```

Comment:
Nit: Consider renaming this file to be more specific (e.g.: `DSCConfigurationFileActivationHandler`) since the `CanHandleInternal` is domain specific